lib/Worker.js

```diff
@@ -84,11 +84,6 @@ module.exports = (function(){
                     logger.debug("[%s] Adding entry to list of handled entries", e.entryId);
                     that.handledEntries[e.entryId] = entry;
                     //entry.on('stopped', onLiveEntryStopped.bind(that));
-                    logger.debug("[%s] Removing entry from endlistAppender", e.entryId);
-                    that.endListAppender.removeEntry(e.entryId) // Entry is alive
-                        .catch(function(err){
-                            logger.error("[%s] Error adding entry to endlist appender processing - %s", e.entryId, err.message);
```

Comment:
what if err is null? or a string? you will have an unhandled exception...
talk with Ron, he added a new utility to print errors